lib/mongo/error/invalid_update_document.rb

```diff
@@ -25,17 +25,16 @@ class InvalidUpdateDocument < Error
 
       # The error message.
       #
-      # @since 2.0.0
+      # @deprecated
       MESSAGE = 'Invalid update document provided'.freeze
 
       # Instantiate the new exception.
       #
-      # @example Instantiate the exception.
-      #   Mongo::Error::InvalidUpdateDocument.new
-      #
-      # @since 2.0.0
-      def initialize
-        super(MESSAGE)
+      # @param [ String ] key The invalid key.
+      def initialize(key)
```

Comment:
Same for this added constructor argument.

Reply:
done